### PR TITLE
Group administration page

### PR DESCRIPTION
**Pull request by GabrielVidal1**
Started to add the group "administration" tab.
Added just one column for the name of each manager at the moment.

### CODE REVIEW

**Review by smadbe — COMMENTED** [review]
Few details to be changed. 
Note that you do not handle  the "loading", "error" and "empty" states yet, you should think about it somehow. But maybe you can do this later when I am done with my current changes related to better state management.

File: src/app/modules/group/components/group-manager-list/group-manager-list.component.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <th style="min-width: 110px;">
+      <div class="slanted-header">
+        <div class="slanted-header-content">
+          Name
+        </div>
+      </div>
+    </th>
+    <th style="width: 160px; transform: none;">
+  </tr>
+</ng-template>
```
[the remark this answers is not in the document]
Comment (GabrielVidal1) [reply src/app/modules/group/components/group-manager-list/group-manager-list.component.html]: fixed

File: src/app/modules/group/components/group-manager-list/group-manager-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+@Component({
+  selector: 'alg-group-manager-list',
+  templateUrl: './group-manager-list.component.html',
+  styleUrls: ['./group-manager-list.component.scss']
+})
+export class GroupManagerListComponent implements OnChanges {
+
+  @Input() group: Group;
+
+  cols = [];
```
[the remark this answers is not in the document]
Comment (GabrielVidal1) [reply src/app/modules/group/components/group-manager-list/group-manager-list.component.ts]: I removed it completely, as the columns will not change

File: src/app/modules/group/components/group-manager-list/group-manager-list.component.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+  managers: Manager[] = [];
+
+  constructor(private getGroupManagersService:GetGroupManagersService) {}
```
[the remark this answers is not in the document]
Comment (GabrielVidal1) [reply src/app/modules/group/components/group-manager-list/group-manager-list.component.ts]: fixed

File: src/app/modules/group/components/group-manager-list/group-manager-list.component.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+
+  ngOnChanges(_changes: SimpleChanges) {
+    this.reloadData();
+  }
+
+  private reloadData() {
+    this.getGroupManagersService
+      .getGroupManagers(this.group.id,)
```
[the remark this answers is not in the document]
Comment (GabrielVidal1) [reply src/app/modules/group/components/group-manager-list/group-manager-list.component.ts]: fixed

File: src/app/modules/group/components/group-manager-list/group-manager-list.component.html [hunk 5]
```diff
@@ -1,41 +1,47 @@
...
-        style="text-align: left;width: 160px;padding-left: 0.5rem;"
-        tooltipPosition="top"
-        tooltipStyleClass="tooltip-custom"
-      >
-        {{ rowData.name }}
-      </td>
-    </tr>
-  </ng-template>
-</alg-grid>
+<ng-container *ngIf="state === 'ready'">
```
Comment (smadbe) [inline src/app/modules/group/components/group-manager-list/group-manager-list.component.html:5]: you could have put the `*ngIf="state === 'ready'"` on `alg-section ` without wrapping everything.

**Review by smadbe — APPROVED** [review]

**Pull request merged**